deploy/kube-config/google/heapster.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: heapster
+  namespace: kube-system
+---
+kind: ClusterRoleBinding
```

Comment:
the service account and deployment are logically paired. the RBAC objects are not, since they would not apply if RBAC authz was not in use.